subsys/net/buf.c

```diff
@@ -26,8 +26,8 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define NET_BUF_ERR(fmt, ...) LOG_ERR(fmt, ##__VA_ARGS__)
 #define NET_BUF_WARN(fmt, ...) LOG_WRN(fmt, ##__VA_ARGS__)
 #define NET_BUF_INFO(fmt, ...) LOG_INF(fmt, ##__VA_ARGS__)
-#define NET_BUF_ASSERT(cond) do { if (!(cond)) {			  \
-			NET_BUF_ERR("assert: '" #cond "' failed"); \
+#define NET_BUF_ASSERT(cond) do { if (!(cond)) {                             \
```

Comment:
I would question why/do we still need an adhoc assert just for net_buf. @jukkar? Assuming it's still needed, I'd rather see __ASSERT_COND_INFO() applied here, instead of unconditional change like that. Printing file/line is of course good.

Reply:
> I would question why/do we still need an adhoc assert just for net_buf. @jukkar?

I am ok if we remove these, @jhedberg do you agree?

Reply:
Yes, I'm fine with removing

Reply:
Removed all NET_BUF_ASSERTs and replaced them with __ASSERT_NO_MSG instead.